backend/internal/utils/actions.go

```diff
@@ -0,0 +1,52 @@
+package utils
+
+import (
+	"fmt"
+	"time"
+)
+
+func ActionWithDeadline(deadline *time.Time) string {
```

Comment:
I think the ideal way to implement this would be a python script (in it's own .py file) that takes in arguments, since I could see any kind of formatting done on this file could break the python code here. Since this works, we can keep it for now and log this as a future enhancement